src/pages/Cart/Cart.js

```diff
@@ -0,0 +1,5 @@
+import "./Cart.css";
+
+export default function () {
+  return <div className="cart"></div>;
+}
```

Comment:
Better not to use anonymous function while creating a component. 
It will give a build in issue, and it can lead to complications that can appear.

Reply:
Resolved.